### PR TITLE
webui: Add label name to 'Certificates' section

### PR DESCRIPTION
**Pull request by carma12**
For testing purposes and uniformity, the 'Certificates' label (located under 'Active users' settings ) should also have a 'name' attribute, as seen in other parts of the WebUI.

Fixes: https://pagure.io/freeipa/issue/8946
Signed-off-by: Carla Martinez <carlmart@redhat.com>

### CODE REVIEW

**Comment by flo-renaud** [comment]
@mrizwan93 can you check if this PR unblocks your PR #5945 ? Thanks

**Comment by abbra** [comment]
@carma12 the temporary commit doesn't actually enable the test you added because you haven't symlinked the temp_commit.yaml like it is suggested at the top of the file:
```
#
# Template for temporary test commit
#
# $ ln -sf ipatests/prci_definitions/temp_commit.yaml .freeipa-pr-ci.yaml
#
```

You need to make this change a part of your commit in order to be able to use the temp_commit.yaml.

**Comment by carma12** [comment]
Hi @abbra 
You are right, I completely forgot that step. 
I corrected that but didn't see on time that @mrizwan93 also pushed on this branch. So maybe the tests were executed twice.

**Review by mrizwan93 — APPROVED** [review]
Test is passing, hence approving the PR.

**Comment by carma12** [comment]
Thanks for the review. Can I set the `ack` flag myself or should the reviewers do it? Should we backport this to `ipa-4-10` and `ipa-4-9` also?

File: ipatests/test_webui/test_user.py [hunk 1]
```diff
@@ -347,6 +347,35 @@ def test_certificates(self):
         self.navigate_to_entity(user.ENTITY, 'search')
         self.delete_record(user.PKEY, user.DATA.get('del'))
 
+    @screenshot
+    def test_certificate_serial(self):
+        """Test long certificate serial no
+        Long certificate serial no were shown in scientific notation
+        at user detials page. This test checks that it is no longer shown
```
Comment (flo-renaud) [inline ipatests/test_webui/test_user.py:354]: Typo: details

**Review by flo-renaud — COMMENTED** [review]
Hi,
thanks for the PR, LGTM. @carma12 there is a small typo but you can fix it and remove the temp commit in a single pass. I will ack the PR when prci completes.
Adding ipa-4-9 and ipa-4-10 labels because the fix for long serial numbers (https://pagure.io/freeipa/issue/8754) was done on ipa-4-8 and above.

**Comment by flo-renaud** [comment]
For future ref, link to the successful run of `test_webui_users`: [Details](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/b8ce5dd2-5130-11ed-8255-fa163ed5a785)

**Comment by carma12** [comment]
Hi @flo-renaud 
Thanks for the review. Already fixed the typo and removed the temp commit. Waiting for the tests to finish.

**Comment by carma12** [comment]
master:

* b76bb195a5bb39c2aa58bd912daf69028f6c6203 webui: Add label name to 'Certificates' section
* 746a036c7eab177fd87a37f0515a46419f22c12b ipatests: Test newly added certificate lable
* f15da104542f67a07cbb4430c6936cf8d35b451e webui: Add name to 'Certificates' table

**Pull request closed without merging**